models/place-videos.json

```diff
@@ -56,7 +56,8 @@
         "type": "string",
         "format": "uuid",
         "example": "46e9ea9f-fc42-4759-a81b-4308467b7c35",
-        "description": "The id of the added video."
+        "description": "The id of the added video.",
```

Comment:
Nit: Maybe the `added` here is also a bit weird since every video on the place (when using `GET /places/{id}`) is of course "added".

Reply:
I think this has carried over from some kind of response to a `POST` or `PUT` request to add a video